src/scales/scale.linear.js

```diff
@@ -134,16 +133,22 @@ module.exports = function(Chart) {
 			this.handleTickRangeOptions();
 		},
 		getTickLimit: function() {
```

Comment:
`getTickLimit()` should be fully overridable by custom scales, which would not be anymore the case with this approach.

Reply:
However, this could be done by introducing a private `_computeTickLimit()` method:

```javascript
// linearbase.js
getTickLimit: function() {
    var me = this;
    var tickOpts = me.options.ticks;
    var stepSize = tickOpts.stepSize;
    var maxTicksLimit = tickOpts.maxTicksLimit;
    var maxTicks;

    if (stepSize > 0) {
        maxTicks = Math.ceil(me.max / stepSize) - Math.floor(me.min / stepSize) + 1;
    } else {
        maxTicks = me._computeTickLimit();
        maxTicksLimit = maxTicksLimit || 11;
    }

    if (maxTicksLimit) {
        maxTicks = Math.min(maxTicksLimit, maxTicks);
    }

    return maxTicks;
}
```